syncthing.te

```diff
@@ -0,0 +1,90 @@
+policy_module(syncthing, 1.0.0)
+
+# Declarations
+attribute_role syncthing_roles;
+role syncthing_roles types syncthing_t;
+
+type syncthing_t;
+type syncthing_exec_t;
+# should only ever be execed non-root, but can be started by systemd --user as a service
+init_daemon_domain(syncthing_t, syncthing_exec_t)
+userdom_user_application_domain(syncthing_t, syncthing_exec_t)
+userdom_use_user_ptys(syncthing_t)
```

Comment:
`userdom_use_inherited_user_terminals` doesn't exist in refpolicy.

I'm not sure how crucial this permission is to the runtime of Syncthing. Due to the warning present in `userdom_use_user_terminals`, I am hesitant about this permission:

```
##	However, this also allows the applications to spy
##	on user sessions or inject information into the
##	user session.  Thus, this access should likely
##	not be allowed for non-interactive domains.
```

Is this access okay for a daemon-like program running in the background?